elements/rh-accordion/rh-accordion-header.css

```diff
@@ -145,3 +154,10 @@ span {
   inset-block: 0;
   inset-inline-start: 0;
 }
+
+@media only screen and (max-width: 576px) {
```

Comment:
Maybe a `@container` query here instead of `@media`

Reply:
@zeroedin i tried adding @container but it doesn't work for me. this is the first time i'm using container query.